### ISSUE TITLE
View per-message OMEMO fingerprint

### ISSUE THREAD

**Issue by kousu** [dino/dino#993]
Conversations has a poorly known but very important feature, if you're trying to use OMEMO with checks and balances, and not just as an improvement over the status quo: it can tell you *what device* sent which message.

I'm not talking about XMPP /resources, though those are related I suppose, I mean it will tell you the key that signed the message.

The trick is to long-press on the *contact icon* next to a message, then go to "Contact Details"

![message-details1](https://user-images.githubusercontent.com/987487/107983279-88be9880-6f93-11eb-8bb7-dc8ec38ca7a8.png)

The message's fingerprint ((actually the fingerprint of the device that signed the fingerprint of the message)) is mentioned:

> OMEMO Fingerprint of message

![message-details2](https://user-images.githubusercontent.com/987487/107983280-88be9880-6f93-11eb-8dbb-193c01c629f0.png)

I think Conversations' UI is opaque and counterintuitive here, but at least I can get this information out. Could we talk about a way to see the same information in dino? :)

**Comment by moppman** [comment]
>The trick is to long-press on the contact icon next to a message, then go to "Contact Details"
No need to long-press. If you tap on the contact icon (method 1), you get to the same "Contact details"  page as you would after using the burger menu-->contact details nav. in the conversation view (method 2).
The only difference is that method 1 highlights the fingerprint of the device that sent the message.
But maybe this is a recent addition? I'm on Conversations 2.9.6+pcr.

**Comment by kousu** [comment]
You're right, you can tap or long-press on the contact icon next to the message to see the fingerprint.

The burger menu does not highlight any fingerprint because it's not associated with a specific message.

I'm also on 2.9.6+pcr, but I think this feature has been around for at least a year.
